selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/CodenvyEditor.java

```diff
@@ -212,6 +213,7 @@ public CodenvyEditor(
     String HOVER_POPUP_XPATH =
         "//div[@class='textviewTooltip' and contains(@style,'visibility: visible')]";
     String AUTOCOMPLETE_PROPOSAL_DOC_ID = "gwt-debug-content-assistant-doc-popup";
+    String LANGUAGE_SERVER_RENFACTORING_RENAME_FIELD_CSS = "input.orionCodenvy";
```

Comment:
`input.orionCodenvy` style name looks too general IMHO

Reply:
I agree but it is part of Orion component. With this general name of css locator is simple and reliable. If you have other idea for this - we will apply it